File: live_stream_app/lib/modules/dashboard/model/approve_request.dart
class ApproveRequest {
  ApproveRequest({
      this.streamID, 
      this.userID, 
      this.active,});

  ApproveRequest.fromJson(dynamic json) {
    streamID = json['streamID'];
    userID = json['userID'];
    active = json['active'];
  }
  int? streamID;
  int? userID;
  bool? active;

  Map<String, dynamic> toJson() {
    final map = <String, dynamic>{};
    map['streamID'] = streamID;
    map['userID'] = userID;
    map['active'] = active;
    return map;
  }

}

File: spring-boot-refresh-token-jwt-master/src/main/java/com/bezkoder/spring/security/jwt/payload/request/ApproveRequest.java
package com.bezkoder.spring.security.jwt.payload.request;

import javax.validation.constraints.NotNull;

/**
 * Created by Benjamin Lam on 3/31/2022.
 * SELF EDU
 * binhlam10796@gmail.com
 */
public class ApproveRequest {

    @NotNull
    private long streamID;

    @NotNull
    private long userID;

    @NotNull
    private boolean active;

    public long getStreamID() {
        return streamID;
    }

    public void setStreamID(long streamID) {
        this.streamID = streamID;
    }

    public long getUserID() {
        return userID;
    }

    public void setUserID(long userID) {
        this.userID = userID;
    }

    public boolean isActive() {
        return active;
    }

    public void setActive(boolean active) {
        this.active = active;
    }
}
